docs/giveortake.md

```diff
@@ -1,14 +1,14 @@
 # Archipelago Contribution Guide [![Google Groups](https://img.icons8.com/wired/32/000000/google-groups.png)](https://groups.google.com/forum/#!forum/archipelago-commons)
 
 
-``Archipelago`` welcomes, appreciates and recognizes any type of contribution, from use cases and needs, questions and answers, documentation, DevOps and configurations to ideas, thoughts and even future dreams. And of course PHP code, including fixes and new features. We want to be embracing, helpful, kind and open. We encourage discussions and respect for anyone's opinions, language, gender, style, backgrounds, origins and destinations if those come from the same root values of respect we state here. We follow and support conflict resolution using nothing more than basic common sense and value diversity in all its shapes, forms, colors, epoches, numbers and kinds, with or without labels, including inbetweens and evolving. We always assume we can do better and that you have done a lot. Under this very basic social framework, this is how we hope you can contribute:
+``Archipelago`` welcomes, appreciates, and recognizes any and all type of contribution. This includes input on all use cases and needs, questions or answers, documentation, DevOps, and configurations.  It also includes more general ideas, thoughts and even future dreams! Of course we also invite you to contribute PHP code, including fixes and new features. We want to be embracing, helpful, kind and open. We encourage discussions and always respect anyone's opinions, language, gender, style, backgrounds, origins and destinations if those come from the same root values of respect we state here. We follow and support conflict resolution using nothing more than basic common sense and value diversity in all its shapes, forms, colors, epoches, numbers and kinds, with or without labels, including inbetweens and evolving. We always assume we can do better and that you have done a lot. Under this very basic social framework, this is how we hope you can contribute:
```

Comment:
Just a tiny form thing, we could avoid having twice "includes" in this paragraph. What do you think? All goods pretty good and i really like how this document, is stating so much and also defining somehow a style and a community around common simple, coherent values. Thanks for correcting!

Reply:
yep. i'm gonna spend some more time w/ all of this, thanks